src/types/base.ts

```diff
@@ -17,6 +17,7 @@ export interface BaseTokenInfo extends Partial<BaseTokenBalance> {
   name: string
   decimals: number
   icon: string
+  blockchain: string
```

Comment:
We have chainId and chainName. Can we use them instead of adding new property?

Reply:
No, because the chain name could be used as a label and in uppercase. I don't think we need it at all.

Reply:
I still don't get why we need 3 fields for describing blockchain.